main.go

```diff
@@ -25,6 +25,15 @@ func main() {
 	width := flag.Int("width", 0, "maximum width")
 	height := flag.Int("height", 0, "maximum height")
 
+  interpolationFunctions := map[string]resize.InterpolationFunction{
+    "nearest": resize.NearestNeighbor,
+    "bicubic": resize.Bicubic,
+    "bilinear": resize.Bilinear,
+    "mitchell": resize.MitchellNetravali,
+    "lanczos2": resize.Lanczos2,
+    "lanczos3": resize.Lanczos3,
+  }
+
```

Comment:
whitespace shenanigans again; you should be using hard tabs for golang, and run it through `gofmt` before committing